onnxruntime/core/framework/customregistry.cc

```diff
@@ -9,7 +9,7 @@ common::Status CustomRegistry::RegisterCustomKernel(KernelDefBuilder& kernel_def
 }
 
 common::Status CustomRegistry::RegisterCustomKernel(KernelCreateInfo& create_info) {
```

Comment:
why not make RegisterCustomKernel() take a KernelCreateInfo&& too?

Reply:
was thinking of not changing more places. Any concern to change the left reference to right reference using std::move please?

Reply:
don't have a concern with the std::move(), but i think an rvalue reference parameter expresses a more specific intent to move from, rather than modify.

Reply:
@edgchen1 I agree with you. I share the same feeling that it's a little bit "weird" to use std::move as cast lvalue reference to rvalue reference, but I'll keep it as it will ask for change more function signatures if not, if no strong preference.

Reply:
that's fine with me. maybe we can make this parameter consistent in a future PR.